ethereum/balancer_v2/view_liquidity.sql

```diff
@@ -0,0 +1,344 @@
+                    1,
+                    2,
+                    3
+            ) foo
+    ),
+    calendar AS (
+        SELECT
+            generate_series(
+                '2021-04-21' :: timestamp,
+                CURRENT_DATE,
+                '1 day' :: INTERVAL
+            ) AS DAY
+    ),
+    cumulative_usd_balance AS (
+        SELECT
+            c.day,
+            b.pool,
+            b.token,
+            cumulative_amount / 10 ^ t.decimals * COALESCE(p1.price, p2.price, 0) AS amount_usd
+        FROM
+            calendar c
+            LEFT JOIN cumulative_balance b ON b.day <= c.day
+            AND c.day < b.day_of_next_change
+            LEFT JOIN erc20.tokens t ON t.contract_address = b.token
+            LEFT JOIN prices p1 ON p1.day = b.day
+            AND p1.token = b.token
+            LEFT JOIN dex_prices p2 ON p2.day <= c.day
+            AND c.day < p2.day_of_next_change
+            AND p2.token = b.token
+    ),
+    pools_tokens_weights AS (
+        SELECT
+            c."poolId" AS pool_id,
+            unnest(cc.tokens) AS token_address,
+            unnest(cc.weights) / 1e18 AS normalized_weight
+        FROM
+            balancer_v2."Vault_evt_PoolRegistered" c
+            INNER JOIN balancer_v2."WeightedPoolFactory_call_create" cc ON c.evt_tx_hash = cc.call_tx_hash
+        UNION
+        ALL
+        SELECT
+            c."poolId" AS pool_id,
+            unnest(cc.tokens) AS token_address,
+            unnest(cc.weights) / 1e18 AS normalized_weight
+        FROM
+            balancer_v2."Vault_evt_PoolRegistered" c
+            INNER JOIN balancer_v2."WeightedPool2TokensFactory_call_create" cc ON c.evt_tx_hash = cc.call_tx_hash
+    ),
+    pool_liquidity_estimates AS (
+        SELECT
+            b.day,
+            b.pool,
+            SUM(b.amount_usd) / COALESCE(SUM(w.normalized_weight), 1) AS liquidity
+        FROM
+            cumulative_usd_balance b
+            LEFT JOIN pools_tokens_weights w ON b.pool = w.pool_id
+            AND b.token = w.token_address
+        GROUP BY
+            1,
+            2
+    ),
+    balancer_liquidity AS (
+        SELECT
+            b.day,
+            b.pool,
+            pool_symbol,
+            token AS token_address,
+            symbol AS token_symbol,
+            coalesce(amount_usd, liquidity * normalized_weight) AS usd_amount
+        FROM
+            pool_liquidity_estimates b
+            LEFT JOIN cumulative_usd_balance c ON c.day = b.day
+            AND c.pool = b.pool
+            LEFT JOIN pools_tokens_weights w ON b.pool = w.pool_id
+            AND w.token_address = c.token
+            LEFT JOIN erc20.tokens t ON t.contract_address = c.token
+            LEFT JOIN pool_labels p ON p.pool_id = SUBSTRING(b.pool :: text, 0, 43) :: bytea
+    )
+    SELECT
+        *
+    FROM
+        balancer_liquidity
+);
+
+CREATE UNIQUE INDEX IF NOT EXISTS balancer_v2_view_liquidity_idx ON balancer_v2.view_liquidity (DAY, pool_id, token_address);
```

Comment:
I'm getting a `psql:view_liquidity.sql:325: ERROR:  column "pool_id" does not exist` error here. Could you double check and commit a fix? I ran the query by itself and saw the `pool_id` field was indeed missing.